src/pages/elected-reps.js

```diff
@@ -3,14 +3,137 @@ import { Link } from "gatsby"
 
 import Layout from "../components/layout"
 import SEO from "../components/seo"
+import axios from "axios"
+import Representative from "../components/representative"
 
-const SecondPage = () => (
-  <Layout>
-    <SEO title="Page two" />
-    <h1>Hi from the second page</h1>
-    <p>Welcome to page 2</p>
-    <Link to="/">Go back to the homepage</Link>
-  </Layout>
-)
+export default class ElectedReps extends React.Component {
+  state = {
+    address: "",
+    officials: [],
+    offices: [],
+    error: "", // Question: should this be part of the state?
```

Comment:
I think it works well! Makes it easy to keep the error message in view until we explicitly reset it.